AutoRest/Generators/CSharp/CSharp/CSharpCodeGenerator.cs

```diff
@@ -6,6 +6,8 @@
 using System.Threading.Tasks;
 using Microsoft.Rest.Generator.ClientModel;
 using Microsoft.Rest.Generator.CSharp.Templates;
+using System.Collections.Generic;
```

Comment:
@yugangw-msft From a previous iteration of the code -- I missed removing them.  Removed them now.